src/index.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-disable */
+import ReactDOM from 'react-dom/client';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import TodoContainer from './functionBased/components/TodoContainer';
+import About from './functionBased/pages/About';
+import NotMatch from './functionBased/pages/NotMatch';
+import Navbar from './functionBased/components/Navbar';
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route exact path="/" element={<TodoContainer />} />
+        <Route path="/about" element={<About />} />
```

Comment:
You class base show errors.
![image](https://user-images.githubusercontent.com/13853450/178068103-35e218a4-3846-42a8-a4ea-514f4c015db8.png)